algebird-core/src/main/scala/com/twitter/algebird/CountMinSketch.scala

```diff
@@ -780,7 +821,7 @@ case class TopNLogic[K](heavyHittersN: Int) extends HeavyHittersLogic[K] {
 
   require(heavyHittersN > 0, "heavyHittersN must be > 0")
 
-  override def purgeHeavyHitters(cms: CMS[K])(hitters: HeavyHitters[K]): HeavyHitters[K] = {
+  def purgeHeavyHitters(cms: CMS[K])(hitters: HeavyHitters[K]): HeavyHitters[K] = {
```

Comment:
Eh I kinda love-hate the override keyword.  It doesn't require that the base class/trait actually implements the method (i.e., so that we know if super is interesting).  Then again, plain def doesn't require that the base class/trait declares the method either.  I'll put it back.